src/amg/setup.py

```diff
@@ -16,10 +16,8 @@
 
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
-
 VERSION = '0.1.3'
```

Comment:
Do you need to update version for release?

Reply:
Let us do it later so to align with other release activity